src/api/spec/fixtures/files/rpmlint_log_extractor_log

```diff
@@ -1,21 +1,21 @@
-HTTP/1.1 200 OK
-X-Frame-Options: SAMEORIGIN
-X-XSS-Protection: 1; mode=block
-X-Content-Type-Options: nosniff
-X-Download-Options: noopen
-X-Permitted-Cross-Domain-Policies: none
-Referrer-Policy: strict-origin-when-cross-origin
-X-Opensuse-APIVersion: 2.10.50
-Content-Type: text/plain
-Content-Disposition: inline
-Content-Transfer-Encoding: binary
-X-Opensuse-Runtimes: {"view":null,"db":16.376543004065752,"backend":0}
-Cache-Control: no-cache
-X-Request-Id: f030558c-aed3-4f72-8113-d434b8e40ba7
-X-Runtime: 0.900190
-Server-Timing: sql.active_record;dur=34.83, start_processing.action_controller;dur=0.18, instantiation.active_record;dur=24.52, cache_read.active_support;dur=1.77, cache_write.active_support;dur=0.60, send_file.action_controller;dur=0.36, process_action.action_controller;dur=662.78
-Content-Length: 34454
-
+HTTP/1.1 200 OK
+X-Frame-Options: SAMEORIGIN
+X-XSS-Protection: 1; mode=block
+X-Content-Type-Options: nosniff
+X-Download-Options: noopen
+X-Permitted-Cross-Domain-Policies: none
+Referrer-Policy: strict-origin-when-cross-origin
+X-Opensuse-APIVersion: 2.10.50
+Content-Type: text/plain
+Content-Disposition: inline
+Content-Transfer-Encoding: binary
+X-Opensuse-Runtimes: {"view":null,"db":16.376543004065752,"backend":0}
+Cache-Control: no-cache
+X-Request-Id: f030558c-aed3-4f72-8113-d434b8e40ba7
+X-Runtime: 0.900190
+Server-Timing: sql.active_record;dur=34.83, start_processing.action_controller;dur=0.18, instantiation.active_record;dur=24.52, cache_read.active_support;dur=1.77, cache_write.active_support;dur=0.60, send_file.action_controller;dur=0.36, process_action.action_controller;dur=662.78
+Content-Length: 34454
+
```

Comment:
I don't think these space only chnages are needed... Are them?

Reply:
Ugh, this was unintentional. Happened automatically because of IDE. I've reverted it

Reply:
All this change is happening because of some weird encoding change in IDE. The only important change here is that I have added an invalid byte sequence in this file